Vagrantfile

```diff
@@ -66,6 +58,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       inventory_path = "provisions/hosts.vagrant"
   end
 
+  # Ensure Jenkins SSH keys exist
+  system('if [ ! -f /tmp/cccp-jenkins.key ]; then ssh-keygen -t rsa -N "" -f /tmp/cccp-jenkins.key; fi')
+
   config.vm.define "master" do |master|
     master.vm.hostname = "cccp"
     master.vm.network :private_network, ip: "192.168.100.100"
```

Comment:
IMO, IP address should be defined in a constant and reused.  I have seen cases whereI create multiple VMs and the IP address conflicts.

Defined in a variable like `VM_IP = "192.168.100.200` atop Vagrantfile, the variable can be re-used at all places like "#{VM_IP}".

"PUBLIC_ADDRESS" variable name is used for such use cases in ADB vagrantfiles.